dss-api.yml

```diff
@@ -702,6 +702,118 @@ paths:
                     enum: [unhandled_exception, illegal_arguments, bundle_already_exists]
                 required:
                   - code
+  /bundles/{uuid}/checkout:
+    post:
+      summary: Check out a bundle to HCA-owned cloud storage
+      description: |
+        User makes a request for a bundle version to be extracted. Storage system makes an attempt to validate
```

Comment:
"Storage service" not "storage system"

Reply:
@kislyuk I used existing terminology see at the top of swagger file:
swagger: '2.0'
info:
  title: HCA DCP DSS API
  description: Human Cell Atlas Data Coordination Platform Data **Storage System** API

Reply:
Got it, my bad.